pyspider/run.py

```diff
@@ -368,6 +368,9 @@ def webui(ctx, host, port, cdn, scheduler_rpc, fetcher_rpc, max_rate, max_burst,
 @cli.command()
 @click.option('--phantomjs-path', default='phantomjs', help='phantomjs path')
 @click.option('--port', default=25555, help='phantomjs port')
+@click.option('--proxy', default=None, help='phantomjs proxy')
+@click.option('--proxy-type', default=None, help='phantomjs proxy-type [http|socks5|none]')
+@click.option('--ignore-ssl-errors', default=None, help='phantomjs ignore-ssl-errors')
 @click.option('--auto-restart', default=False, help='auto restart phantomjs if crashed')
 @click.pass_context
 def phantomjs(ctx, phantomjs_path, port, auto_restart):
```

Comment:
You might need to add proxy, proxy_type, ignore_ssl_errors to parameter list here.